charts/karmada-operator/README.md

```diff
@@ -0,0 +1,71 @@
+# karmada-operator
```

Comment:
I'd like to invite @windsonsea to review the doc.

Reply:
This README looks good to me now.